src/representations/representation_manager_sorted_coulomb.hh

```diff
@@ -65,13 +65,14 @@ namespace rascal {
     // Enum class defining the several possible sorting options of the Coulomb
     // Matrix
     enum class CMSortAlgorithm {
-      Distance, // sort according to the distance from the central atom
-      RowNorm, // sort according to the norm of the coulomb matrix rows
+      Distance,  // sort according to the distance from the central atom
```

Comment:
Ideally there shouldn't be any cosmetic changes to unrelated files (sorted-coulomb, in a soap pull request)

Reply:
well this is automatic